runtime/pavex_macros/src/methods.rs

```diff
@@ -74,6 +77,16 @@ pub fn methods(_metadata: TokenStream, input: TokenStream) -> Result<TokenStream
         method.attrs.extend(new_attributes);
     }
 
+    if no_pavex_method {
+        return Err(syn::Error::new_spanned(
+            &impl_,
```

Comment:
Will this render the whole impl block with the error?
If yes, would it make sense to only render the error on the `#[pavex::methods]` annotation above the impl block?